SubmitReview/ReadMe.md

```diff
@@ -0,0 +1,50 @@
+# SubmitReview
+
+Move fixed Jira tickets from one state to another using the `Request Changes` transitions.
```

Comment:
Please add a sentence that will say that this should move the issue from IN REVIEW to IN PROGRESS

Reply:
My understanding is that we will have more states with the transition `Request Changes,` so we might move the tickets from APPROVED to IN PROGRESS.
Do we still want to mention that it should move only from IN REVIEW to IN PROGRESS?